.github/workflows/dependabot-pr.yml

```diff
@@ -1,7 +1,7 @@
 name: Dependabot PR
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
i know `pull_request_target` is kinda dangerous; 

just to be sure, this action doesn't run on community prs, thanks to our ci-gate mechanism?

Reply:
afaik, this one will run on the base reference, but yeah, I'm a little bit concerned about it because we have used some untrusted inputs in the pipeline. let me do some hack to it. tbh, I'm now leaning to have a secret bot to update this one instead 🙈

Reply:
btw, I can block it to all community prs. it's not from ci-gate. it's the github action built-in mechanism.

Reply:
yes, you're right. I found some methods can successfully attack it. thank you for bumping this issue to make me scan the code carefully again!

btw, I found another method to set up for dependabot. I've removed this one!

Reply:
cool. could you update the pr description, then?